### ISSUE TITLE
are there any special privacy issues around the nonced claim?

### ISSUE THREAD

**Issue by mcr** [ietf-rats-wg/architecture#399]
The challenger could inject identifying information into the nonce claim.
Do you need to say something specific about this?

**Comment by dthaler** [comment]
I think this discussion belongs in the document that defines the nonce claim (EAT) not the arch doc per se.  Same for any other specific claims that might have similar issues.

**Comment by gmandyam** [comment]
>I think this discussion belongs in the document that defines the nonce claim (EAT) not the arch doc per se. Same for any other specific claims that might have similar issues.

I think if a discussion regarding privacy for replay protection mechanisms is required, then all documents that EAT inherits from should address this.  EAT already inherits the jti/cti claims from RFC 7519/8392 respectively.  There is no specific privacy discussion in those documents regarding these claims.  Moreover, the cti/jti claims can be client-originated (potentially internal to entity) and can certainly be used for extraction of PII by bad actors, while a nonce is expected to be verifier-generated (i.e. external to entity).

In short, a privacy discussion should cover all anti-replay mechanisms that are possible with CWT/JWT's, not just the claims introduced in EAT.  In that regards, the EAT document may not be the best place for this discussion.

**Comment by thomas-fossati** [comment]
> I think this discussion belongs in the document that defines the nonce claim (EAT) not the arch doc per se. Same for any other specific claims that might have similar issues.

The reason I brought the issue here is that the nonce is introduced in [Section 10.2 of the architecture](https://datatracker.ietf.org/doc/html/draft-ietf-rats-architecture#section-10.2) and since all RATS documents are supposed to inherit from here, it sounded like a good place to discuss the point once and let downstream docs reference it as many times as needed.

Another convenient place where the privacy discussion can be factored out could be [Section 7.1 of the interaction models](https://datatracker.ietf.org/doc/html/draft-ietf-rats-reference-interaction-models-05#section-7.1).

BTW, I have raised a related issue [in the DAA repo](https://github.com/ietf-rats-wg/draft-ietf-rats-daa/issues/1).

**Comment by mcr** [comment]
https://github.com/ietf-rats-wg/eat/pull/164/files
